[2023-09-23T14:11:23.506+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: tests_dag.test_postgres_to_redis manual__2023-09-23T14:11:09.855644+00:00 [queued]>
[2023-09-23T14:11:23.545+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: tests_dag.test_postgres_to_redis manual__2023-09-23T14:11:09.855644+00:00 [queued]>
[2023-09-23T14:11:23.546+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-09-23T14:11:23.642+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): test_postgres_to_redis> on 2023-09-23 14:11:09.855644+00:00
[2023-09-23T14:11:23.653+0000] {standard_task_runner.py:57} INFO - Started process 1042 to run task
[2023-09-23T14:11:23.665+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'tests_dag', 'test_postgres_to_redis', 'manual__2023-09-23T14:11:09.855644+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmpqz5gl3lw']
[2023-09-23T14:11:23.675+0000] {standard_task_runner.py:85} INFO - Job 27: Subtask test_postgres_to_redis
[2023-09-23T14:11:23.887+0000] {task_command.py:415} INFO - Running <TaskInstance: tests_dag.test_postgres_to_redis manual__2023-09-23T14:11:09.855644+00:00 [running]> on host 4ef300bfa8d2
[2023-09-23T14:11:24.175+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='tests_dag' AIRFLOW_CTX_TASK_ID='test_postgres_to_redis' AIRFLOW_CTX_EXECUTION_DATE='2023-09-23T14:11:09.855644+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-09-23T14:11:09.855644+00:00'
[2023-09-23T14:11:24.177+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-09-23T14:11:24.178+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'pytest /opt/***/dags/dag_scripts/tests/test_postgres_to_redis.py']
[2023-09-23T14:11:24.201+0000] {subprocess.py:86} INFO - Output:
[2023-09-23T14:11:28.564+0000] {subprocess.py:93} INFO - ============================= test session starts ==============================
[2023-09-23T14:11:28.576+0000] {subprocess.py:93} INFO - platform linux -- Python 3.8.18, pytest-7.4.2, pluggy-1.3.0
[2023-09-23T14:11:28.578+0000] {subprocess.py:93} INFO - rootdir: /opt/***/dags/dag_scripts/tests
[2023-09-23T14:11:28.580+0000] {subprocess.py:93} INFO - plugins: anyio-4.0.0
[2023-09-23T14:11:28.580+0000] {subprocess.py:93} INFO - collected 1 item
[2023-09-23T14:11:28.581+0000] {subprocess.py:93} INFO - 
[2023-09-23T14:11:28.667+0000] {subprocess.py:93} INFO - ../../opt/***/dags/dag_scripts/tests/test_postgres_to_redis.py .     [100%]
[2023-09-23T14:11:28.669+0000] {subprocess.py:93} INFO - 
[2023-09-23T14:11:28.669+0000] {subprocess.py:93} INFO - ============================== 1 passed in 3.04s ===============================
[2023-09-23T14:11:29.240+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-09-23T14:11:29.580+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=tests_dag, task_id=test_postgres_to_redis, execution_date=20230923T141109, start_date=20230923T141123, end_date=20230923T141129
[2023-09-23T14:11:29.709+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-09-23T14:11:29.736+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
